client/galaxy/scripts/onload.js

```diff
@@ -170,11 +170,27 @@ $(document).ready( function() {
     if (et){
         et = TOURS.hooked_tour_from_data(et);
         if (et && et.steps){
-            var tour = new Tour(_.extend({
-                steps: et.steps,
-            }, TOURS.tour_opts));
-            tour.init();
-            tour.restart();
+            var in_iframe;
+            try {
+                if (window.self !== window.top){
+                    in_iframe = true;
+                } else {
+                    in_iframe = false;
+                }
+            } catch (e) {
+                in_iframe = false;
+            }
+            if (!in_iframe){
```

Comment:
Replacing 11 lines of `var,try,if,else,catch,if` with a single `if,||` seems like a very good deal to me.

Also I think you can do `if(self==top)` to check that you are not in an (i)frame